tools/aa/commands/WorkspaceCommand.java

```diff
@@ -89,7 +89,10 @@ private boolean createWorkspace() {
       fileUtils.mkdirs(workspacePath);
       try {
         fileUtils.copyDirectoryToDirectory(
-            fileUtils.joinPaths(config.getBasePath(), "head"), workspacePath, "^bazel-.*$");
+            fileUtils.joinPaths(config.getBasePath(), "head"),
+            workspacePath,
+            "^bazel-.*$",
+            "startup-os/tools/local_server/web_login/node_modules");
```

Comment:
Let's ignore all `node_modules`, so we don't need to remember to update this list.

Reply:
Done.